libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -276,6 +276,15 @@ def __build_clangsa_config_handler(args, context):
     # Read clang-sa checkers from the config file.
     clang_sa_checkers = context.default_checkers_config.get(CLANG_SA +
                                                             '_checkers')
+
+    if args.collect_stats:
+        stat_checkers = []
+        for checker in checkers:
+            if checker[0].find("statisticsCollector") != -1:
+                stat_checkers.append({checker[0]: True})
+        LOG.debug("Enabling Statistics Checkers")
+        print(stat_checkers)
```

Comment:
Does `print` like this even work in CodeChecker?

Should this list be printed to the user all the time?